tests/test_cnes_sonarqube.py

```diff
@@ -0,0 +1,238 @@
+            ("SonarJava", "5.13.1 (build 18282)"),
+            ("SonarKotlin", "1.5.0 (build 315)"),
+            ("SonarPHP", "3.2.0.4868"),
+            ("SonarPython", "1.14.1 (build 3143)"),
+            ("SonarQube CNES Export Plugin", "1.2"),
+            ("SonarQube CNES Python Plugin", "1.3"),
+            ("SonarQube CNES Report", "3.2.2"),
+            ("SonarRuby", "1.5.0 (build 315)"),
+            ("SonarScala", "1.5.0 (build 315)"),
+            ("SonarTS", "1.9 (build 3766)"),
+            ("SonarVB", "7.15 (build 8572)"),
+            ("SonarXML", "2.0.1 (build 2020)"),
+            ("Svn", "1.9.0.1295")
+        )
+        sonar_plugins = requests.get(f"{self.SONARQUBE_URL}/api/plugins/installed").json()['plugins']
+        installed_plugins = { plugin['name']: plugin['version'] for plugin in sonar_plugins }
+        for name, version in required_plugins:
+            # Hint: if this test fails, one or more plugins may be missing or installed with an outdated version
+            assert installed_plugins[name] == version
+
+    def test_check_qg(self):
+        """As a SonarQube user, I want the SonarQube server to have the CNES
+        Quality Gate configured and set as default so that I can use it.
+        """
+        quality_gates = requests.get(f"{self.SONARQUBE_URL}/api/qualitygates/list").json()['qualitygates']
+        cnes_quality_gates = [ gate for gate in quality_gates if gate['name'] == "CNES" ]
+        # Hint: if one of these tests fails, the CNES Quality Gate may not have been added correctly, check the container logs
+        assert cnes_quality_gates # not empty
+        assert cnes_quality_gates[0]['isDefault']
+
+    def test_check_qp(self):
+        """As a SonarQube user, I want the SonarQube server to have the
+        CNES Quality Profiles available so that I can use them.
+        """
+        required_quality_profiles = (
+            "CNES_JAVA_A",
+            "CNES_JAVA_B",
+            "CNES_JAVA_C",
+            "CNES_JAVA_D",
+            "CNES_PYTHON_A",
+            "CNES_PYTHON_B",
+            "CNES_PYTHON_C",
+            "CNES_PYTHON_D",
+            "CNES_CPP_A",
+            "CNES_CPP_B",
+            "CNES_CPP_C",
+            "CNES_CPP_D",
+            "CNES_C_A",
+            "CNES_C_B",
+            "CNES_C_C",
+            "CNES_C_D",
+            "CNES_C_EMBEDDED_A",
+            "CNES_C_EMBEDDED_B",
+            "CNES_C_EMBEDDED_C",
+            "CNES_C_EMBEDDED_D"
+        )
+        quality_profiles = requests.get(f"{self.SONARQUBE_URL}/api/qualityprofiles/search").json()['profiles']
+        cnes_quality_profiles = [ qp['name'] for qp in quality_profiles if re.match(r'CNES_\w+_[ABCD]', qp['name']) ]
+        for profile in required_quality_profiles:
+            # Hint: if this test fails, one or more Quality Profiles may be missing, check the container logs
+            assert profile in cnes_quality_profiles
+
+    def test_eus_admin(self):
+        """Evil User Story:
```

Comment:
Codacy found an issue: [1 blank line required between summary line and description (found 0) (D205)](https://app.codacy.com/gh/cnescatlab/sonarqube/pullRequest?prid=6252252)